pkgs/development/libraries/libexecinfo/30-linux-makefile.patch

```diff
@@ -0,0 +1,44 @@
+--- Makefile.orig
++++ Makefile
+@@ -23,24 +23,25 @@
+ # SUCH DAMAGE.
+ #
+ # $Id: Makefile,v 1.3 2004/07/19 05:19:55 sobomax Exp $
++#
++# Linux Makefile by Matt Smith <mcs@darkregion.net>, 2011/01/04
+ 
+-LIB=	execinfo
++CC=cc
```

Comment:
What's this? sounds like we should use `CC_FOR_BUILD` and not set it.

Reply:
I'll see if these patches can't be fetched too, while I'm at it, must've missed them.